### PR TITLE
Ramon audain

### PR DESCRIPTION
**Pull request by raaudain**

### CODE REVIEW

File: data/seeds/03-category.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex("Category").truncate()
+    .then(function () {
+      
+      let num = 1;
+      // Inserts seed entries
+      return knex("Category").insert([
+        {id: num++, category: "Animal Products", loc_id: Math.floor(Math.random()*54)+1},
```
Comment (Eiosx) [inline data/seeds/03-category.js]: I don't categories themselves need a location id. Just specific items being sold. Categories should persist for all locations

File: data/migrations/20191221164629_user-tables.js [hunk 2]
```diff
@@ -3,52 +3,87 @@ exports.up = function(knex) {
...
+                .notNullable()
+                .references("id")
+                .inTable("Location")
+                .onUpdate("CASCADE")
+                .onDelete("CASCADE");
+        })
+        .createTable("Category", table => {
+            table.increments();
+
+            table.integer("loc_id")
```
Comment (Eiosx) [inline data/migrations/20191221164629_user-tables.js]: Don't think category needs a location id.

File: data/seeds/03-category.js (hunk 1 above)
Comment (Eiosx) [reply data/seeds/03-category.js]: Also these foreign keys seem to be entirely random and not reference an actual location

Comment (raaudain) [reply data/seeds/03-category.js]: In order to get the average price on an item based on location, I had to connect category to location.  The request URL looks like this:

/api/location/id_of_the_location/category/id_of_the_category/items/avg

Looks odd, but it works.

Comment (raaudain) [reply data/seeds/03-category.js]: The foreign keys references the location ids.  There are 54 countries so I randomly generated ids.  This is dummies info for testing.

**Pull request merged**